src/layout/List/ListSummary.tsx

```diff
@@ -12,12 +18,70 @@ type ListComponentSummaryProps = {
   componentNode: LayoutNode<'List'>;
   emptyFieldText?: string;
 };
+type Row = Record<string, string | number | boolean>;
 
 export const ListSummary = ({ componentNode, isCompact, emptyFieldText }: ListComponentSummaryProps) => {
   const displayData = componentNode.def.useDisplayData(componentNode);
   const validations = useUnifiedValidationsForNode(componentNode);
   const errors = validationsOfSeverity(validations, 'error');
-  const title = useNodeItem(componentNode, (i) => i.textResourceBindings?.title);
+  const title = useNodeItem(
+    componentNode,
+    (i) => i.textResourceBindings?.summaryTitle || i.textResourceBindings?.title,
+  );
+
+  const { tableHeaders, dataModelBindings } = useNodeItem(componentNode);
+  const { formData } = useDataModelBindings(dataModelBindings, DEFAULT_DEBOUNCE_TIMEOUT, 'raw');
+
+  const displayRows: Row[] = [];
+  (formData?.saveToList as Row[])?.forEach((row: Row) => {
+    const { altinnRowId: _, ...rest } = row;
+    displayRows.push(rest);
+  });
```

Comment:
```suggestion
  const displayRows = (formData?.saveToList as Row[]).map((row: Row) => {
    const { altinnRowId: _, ...rest } = row;
    return rest;
  });
```
Ville også sett på hvordan faktisk verifisere typene i stedet for å overskrive TS med `as Row[]` hvis det er mulig uten større omskrivninger.